apps/socket/src/handlers/handler.ts

```
@@ -0,0 +1,84 @@
+	constructor(
+		protected readonly socket: Socket,
+		protected readonly io: Server
+	) {}
+
+	protected abstract readonly nameSpace: string;
+	protected readonly subNameSpace: string | null = null;
+
+	protected abstract readonly events: string[];
+
+	private _currentEvent?: string;
+	private _currentCallback?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)